README.md

```diff
@@ -49,22 +50,26 @@ component_hls = room_api.add_component(room.id, options=ComponentOptionsHLS())
 # ComponentHLS(id='5f062447-a9f7-45ed-8d1b-511f77dc78ae', properties=ComponentPropertiesHLS(low_latency=False, persistent=False, playable=False, subscribe_mode=<ComponentPropertiesHLSSubscribeMode.AUTO: 'auto'>, target_window_duration=None), type='hls')
 ```
 
-All methods in `RoomApi` may raise one of the exceptions deriving from `fishjam.errors.HTTPError`. They are defined in `fishjam.errors`.
+All methods in `RoomApi` may raise one of the exceptions deriving from `fishjam.errors.HTTPError`. They are defined in
+`fishjam.errors`.
 
 #### Notifier
 
 Notifier allows for receiving real-time updates from the Fishjam Server.
 
-You can read more about notifications in the [Fishjam Docs](https://fishjam-cloud.github.io/fishjam-docs/next/getting_started/notifications).
+You can read more about notifications in the
+[Fishjam Docs](https://fishjam-cloud.github.io/fishjam-docs/next/getting_started/notifications).
```

Comment:
```suggestion
[Fishjam Docs](https://fishjam.io/documentation/guide/server#listening-to-events).
```

Reply:
We want to use `docs.fishjam.io` soon, so lets update it everywhere once it will be ready